apps/web/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,22 @@
+import type { ReactNode } from "react";
+
+export function Tooltip({
+  show,
+  children,
+}: {
+  show: boolean;
+  children: ReactNode;
```

Comment:
Could we create a type/interface to keep consistency with other components?

Reply:
Done [48d4eaa](https://github.com/Blobscan/blobscan/pull/478/commits/48d4eaaf24fc850b28c20d03f961bfe6259ab92f)